app/services/GuidanceService.scala

```diff
@@ -20,41 +20,42 @@ import connectors.GuidanceConnector
 import javax.inject.{Inject, Singleton}
 import models.ui.Page
 import uk.gov.hmrc.http.HeaderCarrier
-
 import scala.concurrent.{ExecutionContext, Future}
+import repositories.SessionRepository
```

Comment:
IntelliJ is reporting some trailing white space.

Reply:
will strip